charts/rstudio-connect/values.yaml

```diff
@@ -335,6 +335,8 @@ config:
     DataDir: /var/lib/rstudio-connect
   Scheduler:
     InitTimeout: 5m
+  Quarto:
```

Comment:
We should configure a Quarto interpreter location for when this chart is used with local execution. The current images should have `/opt/quarto/1.3.340/bin/quarto`, but that will shift to 1.4.X after the Connect 2024.04.0 release.

We should also consider `Python.Enable=true`. Unfortunately, if this chart is used with local execution, that means that we need to configure at least one Python interpreter. If Python is enabled and there are no interpreters, Connect will fail to start.

CC @dbkegley - do you have a different opinion? I'm surprised that the chart doesn't enable Python already!

Reply:
NEWS should call out that Quarto is enabled.

Reply:
Yes, I did add the news. I know in PPM, putting in a direct path caused issues, as if it isn't updated, it will break. Can you add to the sequence for when Connect version is bumped to make sure the quarto path is checked?

Reply:
Yeah, I think python not being enabled for local execution by default is just an artifact of this chart being heavily influenced by off-host. Since python is already installed in the default Connect image I don't see any problem enabling it for local execution by default, that seems like the right call.

Reply:
The latest commit leaves the default values alone now and only enabled Quarto like Python when launcher is enabled. Maybe worth a separate PR if we want to consider turning those on for the none OHE case.